src/Native/Unix/System.Native/CMakeLists.txt

```diff
@@ -55,3 +55,7 @@ endif ()
 
 install_library_and_symbols (System.Native)
 install (TARGETS System.Native-Static DESTINATION .)
+
+# We have to cast struct to uint8_t and back
+set_source_files_properties(pal_networkstatistics.c PROPERTIES COMPILE_FLAGS -Wno-cast-align)
```

Comment:
Disable it globally? Seems like a pretty useless warning for the kind of code that System.Native is.